index.html

```diff
@@ -11,12 +11,26 @@
   <body>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <button onclick="showMobileMenu();" type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
       <nav id="desktop-nav">
         <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#projects">Portfolio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
```

Comment:
- [ ] Please kindly delete the envelope icon ✉️  from the mobile menu:
![image](https://user-images.githubusercontent.com/77038610/142716644-271caebf-6a67-4ddb-95d6-d970744ad334.png)
so that your design follows the  [original](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471):
![image](https://user-images.githubusercontent.com/77038610/142716596-1b0ba9a4-1902-46d4-b831-47e9f86c28bc.png)